IDE/MDK5-ARM/Conf/user_settings.h

```diff
@@ -21,21 +21,20 @@
 
 #define NO_MAIN_DRIVER
 #define BENCH_EMBEDDED
-#define NO_DEV_RANDOM
-#define WOLFSSL_USER_CURRTIME
 #define SIZEOF_LONG_LONG 8
 #define NO_WRITEV
 #define NO_DEV_RANDOM
+#define WOLFSSL_IGNORE_FILE_WARN
```

Comment:
I had to define `NO_INLINE` to get it building

Reply:
I did not. That's very odd. Can you please share the errors you were seeing?

Reply:
```
C:/Users/Owner/AppData/Local/Arm/Packs/wolfSSL/wolfSSL/5.6.2/wolfssl/wolfcrypt/test/test.c(36): warning: In file included from...
C:/Users/Owner/AppData/Local/Arm/Packs/wolfSSL/wolfSSL/5.6.2/wolfssl\wolfssl/wolfcrypt/mem_track.h(149): error: unknown type name 'inline'
static WC_INLINE void* TrackMalloc(size_t sz)
       ^
C:/Users/Owner/AppData/Local/Arm/Packs/wolfSSL/wolfSSL/5.6.2/wolfssl\wolfssl/wolfcrypt/types.h(320): note: expanded from macro 'WC_INLINE'
                   #define WC_INLINE inline
                                     ^
C:/Users/Owner/AppData/Local/Arm/Packs/wolfSSL/wolfSSL/5.6.2/wolfssl/wolfcrypt/test/test.c(36): warning: In file included from...
C:/Users/Owner/AppData/Local/Arm/Packs/wolfSSL/wolfSSL/5.6.2/wolfssl\wolfssl/wolfcrypt/mem_track.h(226): error: unknown type name 'inline'
static WC_INLINE void TrackFree(void* ptr)
       ^
C:/Users/Owner/AppData/Local/Arm/Packs/wolfSSL/wolfSSL/5.6.2/wolfssl\wolfssl/wolfcrypt/types.h(320): note: expanded from macro 'WC_INLINE'
                   #define WC_INLINE inline
                                     ^
C:/Users/Owner/AppData/Local/Arm/Packs/wolfSSL/wolfSSL/5.6.2/wolfssl/wolfcrypt/test/test.c(36): warning: In file included from...
C:/Users/Owner/AppData/Local/Arm/Packs/wolfSSL/wolfSSL/5.6.2/wolfssl\wolfssl/wolfcrypt/mem_track.h(237): warning: cast from 'byte *' (aka 'unsigned char *') to 'memoryTrack *' (aka 'struct memoryTrack *') increases required alignment from 1 to 4 [-Wcast-align]
    mt = (memoryTrack*)((byte*)ptr - sizeof(memoryTrack));
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/Users/Owner/AppData/Local/Arm/Packs/wolfSSL/wolfSSL/5.6.2/wolfssl\wolfssl/wolfcrypt/mem_track.h(300): error: unknown type name 'inline'
static WC_INLINE void* TrackRealloc(void* ptr, size_t sz)
       ^
C:/Users/Owner/AppData/Local/Arm/Packs/wolfSSL/wolfSSL/5.6.2/wolfssl\wolfssl/wolfcrypt/types.h(320): note: expanded from macro 'WC_INLINE'
                   #define WC_INLINE inline
                                     ^
C:/Users/Owner/AppData/Local/Arm/Packs/wolfSSL/wolfSSL/5.6.2/wolfssl\wolfssl/wolfcrypt/mem_track.h(791): error: unknown type name 'inline'
static WC_INLINE void StackTrap(void)
       ^
C:/Users/Owner/AppData/Local/Arm/Packs/wolfSSL/wolfSSL/5.6.2/wolfssl\wolfssl/wolfcrypt/types.h(320): note: expanded from macro 'WC_INLINE'
                   #define WC_INLINE inline
                                     ^
```

Reply:
I didn't change the default compiler for the Keil µVision project
```
-xc -std=c90 --target=arm-arm-none-eabi -mcpu=cortex-m23 
```

Reply:
Nope. I'll look if it's getting turned on somewhere for the platform

Reply:
But regardless if inline is an unknown type, might as well disable it
```
 error: unknown type name 'inline'
```

Reply:
Looks like it's complaining about the WC_INLINE in the function declaration in `mem_track.h` and that declaration in the header is not guarded with `WOLFSSL_TRACK_MEMORY`